### PR TITLE
Public Cloud: Increase boot disk size

### PR DESCRIPTION
**Pull request by pdostal**
- Related ticket: [poo#102807](https://progress.opensuse.org/issues/102807)
- Bug example: [sle-15-SP3-EC2-Containers](https://openqa.suse.de/tests/7713114)
- Verification run: [EC2](http://pdostal-server.suse.cz/tests/12889#) [GCE](http://pdostal-server.suse.cz/tests/12887) [Azure](http://pdostal-server.suse.cz/tests/12888)

### CODE REVIEW

**Review by grisu48 — APPROVED** [review]
Simple and nice. LGTM :rocket: I'd like to see a VR for one of the previously failing runs before merging

File: data/publiccloud/terraform/azure.tf [hunk 1]
```diff
@@ -185,6 +185,7 @@ resource "azurerm_linux_virtual_machine" "openqa-vm" {
     name              = "${var.name}-${element(random_id.service.*.hex, count.index)}-osdisk"
     caching              = "ReadWrite"
     storage_account_type = "Standard_LRS"
+    disk_size_gb         = 30
```
Comment (b10n1k) [inline data/publiccloud/terraform/azure.tf:188]: where the 30G comes from? what is the default? looking at the gce i would say that it is 10G. Which 30 makes it too much IMO. Can you check?

Comment (jlausuch) [reply data/publiccloud/terraform/azure.tf:188]: 10GB was the default and we saw it's not enough :)

Comment (pdostal) [reply data/publiccloud/terraform/azure.tf:188]: Right, decreeing to `20G`.

Comment (b10n1k) [reply data/publiccloud/terraform/azure.tf:188]: i still see `30`

Comment (pdostal) [reply data/publiccloud/terraform/azure.tf:188]: Pushed now. However your comment applies only to GCE and EC2 as in Azure the default and smallest possible boot disk size is *30G*.

**Comment by pdostal** [comment]
In *Azure* - the smallest image allowed is *30G8.*:
```
Error: creating Linux Virtual Machine "pdostal-server-c1d2606f7bc71730" (Resource Group "pdostal-server-c1d2606f7bc71730"): compute.VirtualMachinesClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: autorest/azure: Service returned an error. Status=<nil> Code="OperationNotAllowed" Message="The specified disk size 20 GB is smaller than the size of the corresponding disk in the VM image: 30 GB. This is not allowed. Please choose equal or greater size or do not specify an explicit size." Target="osDisk.diskSizeGB"
```
So *GCE* and *EC2* are now set to *20G* and *Azure* is set to *30G*.

**Comment by jlausuch** [comment]
Cool! 
What do you think about adding a `record_info` with output of `df -h` somewhere, maybe in instance_overview?

**Comment by pdostal** [comment]
> Cool! What do you think about adding a `record_info` with output of `df -h` somewhere, maybe in instance_overview?

I for now excluded Azure and am now about to go out to get some fresh air.

The Google and Amazon are ready to be merged.

I'll continue tomorrow.

**Review by jlausuch — APPROVED** [review]

**Review by grisu48 — APPROVED** [review]

**Comment by asmorodskyi** [comment]
too late but still LGTM :)

**Pull request merged**